terraform-environments/hosts-user-data.sh

```diff
@@ -70,12 +66,13 @@ $sudo_path mkdir iamhere2
 $sudo_path chown ${USERNAME}:${USERNAME} /home/${USERNAME}/.ssh/authorized_keys
 $sudo_path chmod 600 /home/${USERNAME}/.ssh/authorized_keys
 
+
 # Update the SSH server configuration
 $sudo_path systemctl reload sshd
 
 $sudo_path mkdir iamhere3
 
-#create aws user
+#create aws user - doesn't do anything since ec2 itself already has admin access
 $sudo_path useradd -m aws_user 
 $sudo_path chown aws_user:aws_user /home/aws_user
 $sudo_path chmod 755 /home/aws_user
```

Comment:
Why are we giving it 755 perms? seems insecure

Reply:
because for some reason otherwise i didn't have access. these were the lowest permissions that worked

Reply:
Where do you access it? If you mean to get in, you should be using the `su` command to switch users